src/components/LinkButton/LinkButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Pressable from '../Clickable/Clickable';
+
+const LinkButton = props => {
+    const openInNewTab = (url) => {
+        const newWindow = window.open(url, '_blank', 'noopener,noreferrer')
+        if (newWindow) newWindow.opener = null
+    }
+
+    return (
+        <Pressable
```

Comment:
Can we change up the styling on the `<LinkButton/>` a bit? Maybe add rounded corners, blue fill, and white text. Also probably would have to decrease the font size a bit.